apps/www/_blog/2023-01-03-launch-week-6-hackathon-winners.mdx

```diff
@@ -0,0 +1,132 @@
+
+## Best Flutter Project
+
+### Winner
+
+[supaquiz - Multiplayer trivia/quiz game](https://github.com/yallurium/supaquiz) - by [@yallurium](https://twitter.com/yallurium).
+
+Supaquiz is a realtime multiplayer trivia/quiz game where you can host or join gmaes with other players remotely.
+
+![supaquiz - Multiplayer trivia/quiz game](/images/blog/lw6-hackathon-winners/supaquiz.png)
+
+### Runner Up
+
+## Best Storage Project
+
+### Winner
+
+[Lookout](https://github.com/luxecraft/lookout) - by [Aman](https://twitter.com/Amanrk28), [Nivas](https://twitter.com/nithsua), [Hari](https://twitter.com/_harisaran_), and [Sabesh](https://twitter.com/sabeshbharathi).
+
+Lookout is an application that lets you discover images from across various media by searching them using their metadata - their properties, content inside the image, textual data found inside the image etc.
+
+![Lookout](/images/blog/lw6-hackathon-winners/lookout.png)
+
+### Runner Up
+
+[Penelope AI](https://github.com/taishikato/Penelope-AI) - by [@taishik\_](https://twitter.com/taishik_).
+
+Penelope AI is an AI writing assistant that can summarize, paraphrase, create stories, and auto-complete from your writing.
+
+![Penelope AI - Unleash the power of your writing with the most sophisticated AI writing assistant.](/images/blog/lw6-hackathon-winners/penelope-ai.png)
+
+## Best Edge Functions Project
+
+### Winner
+
+[Vista](https://github.com/zernonia/vista) - by [@zernonia](https://twitter.com/zernonia).
+
+Vista is a video transcoder that adds subtitles from the audio. The design of the subtitles are customizable to fit the theme of the video.
+
+![Vista](/images/blog/lw6-hackathon-winners/vista.png)
+
+### Runner Up
+
+[Supa Que](https://github.com/suciptoid/supaque) - by [@suciptodev](https://twitter.com/suciptodev).
+
+description
+
+![Supa Que](/images/blog/lw6-hackathon-winners/supaque.png)
+
+## The Prizes
+
+As promised in the [hackathon announcement article](https://supabase.com/blog/launch-week-6-hackathon) the winners of the Best overall project will be granted the extremely limited edition Supabase mechanical keyboard!
+
+![Supabase Keyboard](/images/blog/lw6-hackathon/keyboard.png)
+
+Each members of the winners in other categories will receive a limited edition supaverified swag kit, and the winners of the Best Edge Functions projects will receive swag kits from the Deno team.
+
+![Swag kit](/images/blog/lw6-hackathon/swagkit.png)
+
+The swag kits are limited edition and extremely valuable, so keep them safe! Again, congratulations on all the winners!
+
+## Hang out with us and the winners of the Hackathon
+
+To celebrate the massive success of the Hackathon, we are hosting a Discord Stage Channel with and inviting the winners and everyone who participated. Join us on Thursday, Jan 5th at 7:00 AM PST.
+
+**[Join our Discord](https://discord.supabase.com/)** and **[Set a reminder](https://twitter.com/i/spaces/1LyxBoXywLjKN)** for the event.
```

Comment:
@silentworks Could you update the link for `Set a reminder` once the Discord Stage channel is ready?